src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CallbackProcessor.java

```diff
@@ -51,38 +52,85 @@ public CallbackProcessor(CoreCaseDataApi ccdApi, CcdAuthenticatorFactory authFac
     //TODO these are for the validations of the incoming request and is a WIP
     private List<String> attachCase(String theType,
                                     String anEventId,
-                                    String jurisdiction,
+                                    String exceptionRecordJurisdiction,
                                     String caseTypeId,
                                     String caseRef,
                                     CaseDetails exceptionRecord) {
-        CcdAuthenticator authenticator = authFactory.createForJurisdiction(jurisdiction);
         try {
-            // TODO: use `CaseRetriever`
-            retrieveCase(caseRef, authenticator);
+            attachCase(exceptionRecordJurisdiction, caseRef);
+            return success();
+        } catch (CallbackException e) {
+            return createErrorList(e);
+        }
+    }
+
+    @NotNull
+    private void attachCase(String exceptionRecordJurisdiction, String caseRef) {
+        CcdAuthenticator authenticator = authFactory.createForJurisdiction(exceptionRecordJurisdiction);
+        CaseDetails theCase = getCase(caseRef, authenticator);
+        startAttachScannedDocs(caseRef, authenticator, theCase);
+    }
+
+    @NotNull
+    private List<String> createErrorList(CallbackException e) {
+        String message = e.getMessage();
+        log.error(message, e);
+        return ImmutableList.of(message);
+    }
+
+    @NotNull
+    private List<String> success() {
+        return emptyList();
+    }
+
+    private StartEventResponse startAttachScannedDocs(String caseRef,
+                                                      CcdAuthenticator authenticator,
+                                                      CaseDetails theCase) {
+        try {
+            return startAttachScannedDocs(caseRef, authenticator, theCase.getJurisdiction(), theCase.getCaseTypeId());
+        } catch (FeignException e) {
+            throw error(e, "Internal Error: start event call failed case: %s Error: %s", caseRef, e.status());
+        }
+    }
+
+    private StartEventResponse startAttachScannedDocs(String caseRef,
+                                                      CcdAuthenticator authenticator,
+                                                      String jurisdiction,
+                                                      String caseTypeId) {
+        return ccdApi.startEventForCaseWorker(
+            authenticator.getUserToken(),
+            authenticator.getServiceToken(),
+            authenticator.getUserDetails().getId(),
+            jurisdiction,
+            caseTypeId,
+            caseRef,
+            "attachScannedDocs"
+        );
+    }
+
+    private CaseDetails getCase(String caseRef, CcdAuthenticator authenticator) {
+        try {
+            // TODO: merge with `CaseRetriever` to a consistent api adaptor
```

Comment:
this also moves out to the api wrapper in https://github.com/hmcts/bulk-scan-orchestrator/pull/145